src/D0_modelling_and_optimization.py

```diff
@@ -111,6 +112,15 @@ def warning_asset_type(asset, type, assetGroup):
 
     logging.info("Adding constraints to oemof model...")
     # todo include constraints
+    if "Electrolyser" in dict_model["transformers"]:
+        model_constraints.standby_consumption(
+            local_energy_system,
+            dict_model["busses"]["Electricity bus"],
+            dict_model["busses"]["H2 bus"],
```

Comment:
This should not be referencing the names "Electricity bus" or "H2 bus" but be generalized, ie. sth like "input direction"/"output direction" or how this is named.